packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.types.ts

```diff
@@ -97,9 +97,11 @@ export interface ISpinButtonProps {
 
   /**
    * This callback is triggered when the value inside the SpinButton should be validated.
+   * @param value: The value entered in the SpinButton to validate
+   * @param keyboardEvent: The keyboard event that triggered this validate, if any. (For accessibility)
    * @return {string | void} If a string is returned, it will be used as the value of the SpinButton.
    */
-  onValidate?: (value: string) => string | void;
+  onValidate?: (value: string, keyboardEvent?: React.KeyboardEvent<HTMLElement>) => string | void;
```

Comment:
Should the event be more generic to pass back whatever event caused the validate?

Reply:
I suppose I was making an assumption I shouldn't have that only keyboard events were interesting to consumers. I guess the consumer should make that call. What I didn't like was that now consumers have to type check it. But it sounds like the right thing to do since onBlur indirectly calls it and passes its FocusEvent, and I'm technically passing that event right now (oops).